.github/workflows/docs.yml

```diff
@@ -41,8 +41,11 @@ jobs:
             "</script>"
 
       - name: cargo doc
-        # (CARGO DOC WITH ALL FEATURES IN THIS FORK)
-        run: cargo doc --locked --all-features --no-deps --package ${{ env.MAIN_CRATE }}
+        # Generate docs like docs.rs
+        # IMPORTANT: MUST KEEP IN SYNC with `rustls/Cargo.toml` AS FOLLOWS:
+        # - [package.metadata.docs.rs] section
+        # - FEATURE NAME in sync with CORRESPONDING FEATURE NAME in main crate [features] section
+        run: cargo doc --locked --features unstable-api-all --no-deps --package ${{ env.MAIN_CRATE }}
```

Comment:
I think how a developer should do this for testing purposes needs to be documented.

May be ideal to add some kind of xtask job or admin script to do this, automatically.